website/docs/r/nsxv_dnat.html.markdown

```diff
@@ -16,6 +16,9 @@ modify, and delete destination NATs to map an external IP/port to an internal IP
 ~> **Note:** This resource requires advanced edge gateway. For non-advanced edge gateways please
 use the [`vcd_dnat`](/docs/providers/vcd/r/dnat.html) resource.
 
+!> **Warning:** Do not use older [`vcd_dnat`](/docs/providers/vcd/r/dnat.html) resource with this one
```

Comment:
Should we add such information to TROUBLESHOOTING.md ?

Reply:
Indeed - added a note